src/PerfView/app.config

```diff
@@ -7,4 +7,7 @@
   <startup>
     <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/>
   </startup>
+  <runtime>
+    <AppContextSwitchOverrides value="Switch.System.IO.UseLegacyPathHandling=false;Switch.System.IO.BlockLongPaths=false" />
```

Comment:
`UseLegacyPathHandling` gets set to `true` for container scenarios, so I'm suspicious that this will break long path support.  Does long path support require that `UseLegacyPathHandling==false`?

Reply:
UseLegacyPathHandling=true allows to use also the \\?\ notation which is for some paths an alternate way to reach long path support. But it breaks for UNC file share names. In general it should not be a problem in container scenarios if you are dealing with "normal" paths names. I added it to be more modern and faster because path normalization used to be more expensive when set to false.